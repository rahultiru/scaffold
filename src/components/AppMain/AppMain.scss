.AppMain {
  padding: 24px;
  font-size: 1.6rem;
}

@media (--small-screen) {
  .AppMain {
    padding: 24px 32px;
  }
}

@media (--large-screen) {
  .AppMain {
    padding: 24px 40px;
  }
}
