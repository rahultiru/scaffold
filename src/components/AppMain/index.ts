export { default } from './AppMain';
export * from './AppMain';
