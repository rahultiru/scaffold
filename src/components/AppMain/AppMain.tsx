import React from 'react';
import Routes from 'components/Routes/Routes';

import './AppMain.scss';

/**
 * Main content of the app.
 */
const Main: React.SFC = () => <main className="AppMain" role="main"><Routes /></main>

export default Main;
