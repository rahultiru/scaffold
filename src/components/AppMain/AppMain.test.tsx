import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import AppMain from '.';

describe('AppMain', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<AppMain>foo</AppMain>);
  });

  it('renders', () => {
    expect(component).toMatchSnapshot();
  });
});
