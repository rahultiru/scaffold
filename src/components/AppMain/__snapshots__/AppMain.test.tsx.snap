// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppMain renders 1`] = `
<main
  className="AppMain"
  role="main"
>
  <Block
    textSize="medium"
  >
    foo
  </Block>
</main>
`;
